gadget/gadget_test.go

```diff
@@ -121,6 +121,17 @@ defaults:
       bar: baz
 `)
 
+var mockClassicGadgetMultilineDefaultsYaml = []byte(`
+defaults:
+  system:
+    something: true
+  otheridididididididididididididi:
+    foosnap:
+      multiline: |
```

Comment:
For the sake of pushing the YAML parser, would this be accepted?
```
var mock = []byte(`
defaults:
  other:
    multiline: |
      # this is a multiine string with valid YAML inside
      foo: 2
      bar: "123"
      baz: |
        this is a test
`)
```

Reply:
Good question, hard to say without trying ;). Will give it a shot.